### DISCUSSION TITLE
Падают тесты для статей

### DISCUSSION THREAD

**Discussion by SvetlanaVet** [open-tutorials/cypress#21] [Q&A]
В версии 10 и 11 тесты **should do open article detail page** и **'should do navigate in list by paging** из файла **global-feed.cy.js** работают, в версии 12.4.1 эти тесты падают с одинаковой ошибкой, см. скрины https://disk.yandex.com/i/LmRK5VZIOfqkhQ и https://disk.yandex.com/i/38xmezfbs6SK1w

**Comment by breslavsky** [comment]
@Vladimir4932 please try reproduce the problem.

**Reply by breslavsky** [reply]
@Vladimir4932 any news? 2 week ago!

**Reply by Vladimir-Tarasov** [reply]
I can reproduce it.

**Reply by breslavsky** [reply]
When can we fix it?

**Comment by Vivioza5** [comment]
I reproduced it and fixed it because I experienced it myself. This was a dependency of the Cypress Update API and you can now specify your alias type in the .as() request. [Syntax .as() ](https://docs.cypress.io/api/commands/as#Yields-Icon-namequestion-circle)  and  it's link to notes of update  [12.3.0	Added option type to opt into the pre-12.0.0 behavior.](https://docs.cypress.io/guides/references/changelog#12-3-0)
It's my solution:

```js
cy.get('@randomArticle')
          .find('h1')
          .invoke('text')
          .invoke('trim')
          .as('randomArticleTitle', { type: 'static' });
```

change initialization alias "randomArticleTitle" in all places

**Reply by Vladimir-Tarasov** [reply]
It works. Thanks. 
<img width="504" alt="Screenshot 2023-02-14 at 14 39 42" src="https://user-images.githubusercontent.com/92365254/218755013-8a193e8e-925a-4f37-b3b5-aed5004d68ff.png">
